docs/source/reference/config-proxy.md

```diff
@@ -51,7 +51,7 @@ server {
     listen 80;
     server_name HUB.DOMAIN.TLD;
 
-    # Tell all requests to port 80 to be 302 redirected to HTTPS
+    # Send all requests to port 80 to 302 and redirect to HTTPS
```

Comment:
This configuration line actually returns HTTP code 302 -- it does not redirect traffic from port 80 to port 302.